lib/msal-browser/src/event/BroadcastMessage.ts

```diff
@@ -0,0 +1,16 @@
+import { AuthenticationResult, AuthError, EndSessionRequest } from "@azure/msal-common";
+import { BroadcastEvent } from "./BroadcastEvent";
+import { InteractionType } from "../utils/BrowserConstants";
+import { PopupRequest, RedirectRequest, SilentRequest, SsoSilentRequest } from "..";
+
+export type BroadcastMessage = {
+    type: BroadcastEvent;
```

Comment:
We might want to give this field a more descriptive name, given `interactionType` is also present.

Reply:
Sure. Changed it. Feel free to let me know if you think of a better name than eventType.